src/features/user/CompleteSignup/index.tsx

```diff
@@ -43,6 +43,7 @@ export default function CompleteSignup(): ReactElement | null {
   const { i18n, t, ready } = useTranslation(['editProfile', 'donate']);
   const { setErrors, redirect } = React.useContext(ErrorHandlingContext);
   const [addressSugggestions, setaddressSugggestions] = React.useState([]);
+  const [disable, setDisable] = React.useState<boolean>(false);
```

Comment:
Could you rename `disable` to `isProcessing` to indicate it's purpose better? It indicates that the request is processing, that's why the button is disabled. Update this throughout.

P.S. boolean values should usually be named beginning with `is`, `can`, `should`, `has`. Think of it like asking a question.